topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,653 @@
+1. For heatmap **Heatmap w ggplot** tool is used along with other tabular manipulating tools, such as **Advanced Cut**, **Paste**, **Transpose**, **Replace**, **Multi-Join** and **Add line to file**. All these tools are used to adjust the two of the input tabular files we copied to out history which are `VFs ABRicate Sample Specific Formatted Tabular` and `Sample Specific Contigs FASTA file` from both samples.
+
+2. For the phylogenetic trees, for each bacteria pathogen gene found in the samples we use **ClustalW** for [Multiple Sequence Alignment (MSA)](https://www.sciencedirect.com/topics/medicine-and-dentistry/multiple-sequence-alignment) needed before constructing a phylogenetic tree, for the tree itself we use **FASTTREE** and **Newick Display** to visualize it. Also other table manipulation tools to adjust the other two input files to draw the trees such as, **Cut**, **Split by group**, **Filter sequence by ID**, **Remove beginning** and **bedtools getfasta**.
+
+## Heatmap
+A heatmap is one of the visualization techniques that can give you a complete overview of all the samples together and whether or not a certain value exists. In this tutorial, we use the heatmap to visualize all samples aside and check which common bacteria pathogen genes are found in samples and which is only found in one of them. 
+
+> <question-title></question-title>
+>
+> Now let's see how your heatmap looks like, you can zoom-in and out in your Galaxy history.
+>
+> >    ![Heatmap](./images/heatmap.png)
+>
+> 1. Mention three of the common bacteria pathogen genes found in both samples
+> 2. Mention one that is only found in `Barcode10_Spike2` sample and one that is only found in `Barcode11_Spike2b`
+> 3. How can the differences in the found **VF** bacteria pathogen genes between the two samples be interpreted?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Three of the bacteria pathogen genes with a **VF** identified by the **VFDB** that are common in both samples are with the following accession number: **NP_461810**, **NP_461809** and **NP_459541**
+> > 2. **AAF98391** is only found in Barcode 10 sample and **NP_460360** is only found in Barcode 11 sample
+> > 3. Both samples are spiked with the same species of the pathogen; Salmonella enterica, but not the same strain. `Barcode10_Spike2` sample is spiked with **Enterica** strain and `Barcode11_Spike2b` with Houtenae strain. This can be the main cause of the most similarities and the few difference found bacteria pathogen genes between both of the samples. Other factors such as the time and location of the sampling may cause other differences. By knowing the metadata of the samples inputted for the sub-workflows in real life we can understand what actually happened. We can have samples with no pathogen found then we start detecting genes from the 7th or 8th sample, then we can identify where and when the pathogen entered the host, and stop the cause of that
```

Comment:
```suggestion
> > 3. Both samples are spiked with the same species of the pathogen; _S. enterica_, but not the same strain. `Barcode10_Spike2` sample is spiked with **_S. enterica_** strain and `Barcode11_Spike2b` with _S. enterica subsp. houtenae_ strain. This can be the main cause of the most similarities and the few difference found bacteria pathogen genes between both of the samples. Other factors such as the time and location of the sampling may cause other differences. By knowing the metadata of the samples inputted for the sub-workflows in real life we can understand what actually happened. We can have samples with no pathogen found then we start detecting genes from the 7th or 8th sample, then we can identify where and when the pathogen entered the host, and stop the cause of that
```